qrenderdoc/Code/QRDUtils.cpp

```diff
@@ -442,6 +442,29 @@ QMessageBox::StandardButton RDDialog::messageBox(QMessageBox::Icon icon, QWidget
   return ret;
 }
 
+QMessageBox::StandardButton RDDialog::messageBoxChecked(QMessageBox::Icon icon, QWidget *parent,
+                                                        const QString &title, const QString &text,
+                                                        QCheckBox *checkBox, bool &checked,
+                                                        QMessageBox::StandardButtons buttons,
+                                                        QMessageBox::StandardButton defaultButton)
+{
+  bool isChecked = checked;
```

Comment:
Unimportant nit: if we're already capturing everything by reference (which makes sense here as we're just forwarding the function call in-line to the GUI thread) then do you need this `isChecked`, or can you just assign to `checked` directly?